doc/modules/model_evaluation.rst

```diff
@@ -2354,6 +2354,71 @@ the difference in errors decreases. Finally, by setting, ``power=2``::
 we would get identical errors. The deviance when ``power=2`` is thus only
 sensitive to relative errors.
 
+.. _pinball_loss:
+
+Pinball loss
+------------
+
+The :func:`pinball_loss` function is mostly used to evaluate the predictive
+performance of quantile regression models. The `pinball loss
+<https://en.wikipedia.org/wiki/Quantile_regression#Computation>`_ is
+equivalent to :func:`mean_absolute_error` when the quantile parameter ``alpha``
+is set to 0.5.
+
+.. math::
+
+\text{pinball}(y, \hat{y}) = \frac{1}{n_{\text{samples}}} \sum_{i=0}^{n_{\text{samples}}-1}  \alpha \max(y_i - \hat{y}_i, 0) + (1 - \alpha) \max(\hat{y}_i - y_i, 0)
```

Comment:
@lorentzenchr I did not use `min(..., 0)` because I find it harder to think about double negatives. Here all the terms are positive.

Reply:
That's better. Another way would be to just define (..)_+ below the formula. But I'd say, let's keep it as is.